src/narjure/memory_management/persistence_manager.clj

```diff
@@ -22,18 +51,19 @@
   (shutdown!))
 
 (defn initialise
-  "Initialises actor:
-      registers actor and sets actor state"
+  "Initialises actor: registers actor and sets actor state"
   [aname actor-ref]
   (register! aname actor-ref)
-  (set-state! {:state 0}))
+  (set-state! {:path "d:/persistence/snapshot.nar" :concept-count 0 :received-states 0}))
```

Comment:
Agree, Actor is updated. Still need to add code for setting environment somewhere.